src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -273,6 +273,13 @@ public abstract partial class WebRequestPSCmdlet : PSCmdlet
         [ValidateNotNullOrEmpty]
         public virtual string CustomMethod { get; set; }
 
+        /// <summary>
+        /// Gets or sets the PersistHTTPMethod property.
+        /// </summary>
+        [Parameter]
+        [ValidateSet("All", "300", "301", "302", "303", IgnoreCase = true)]
+        public virtual string[] PersistHTTPMethod { get; set; }
```

Comment:
Since we have `PreserveAuthorizationOnRedirect` we should follow the name schema. What about `PersistHTTPMethodOnRedirect`?

Also do we really need to point exact values? May the parameter be a Switch?

Reply:
What about `PreserveHTTPMethodOnRedirect`?
It could easily be a switch, I followed what was said in the issue.

Reply:
I am ok with `PreserveHTTPMethodOnRedirect`.

Let's start with switch. In any case we will need WG approval. And we have some months before release to get feedback. (Later we can change the parameter type to string[] with "All" default without breaking scripts).

Reply:
Done